pkg/services/baremetal/host/host.go

```diff
@@ -1021,7 +1021,15 @@ func (s *Service) createAutoSetupInput(sshClient sshclient.Client) (autoSetupInp
 	if needsDownload {
 		out := sshClient.DownloadImage(imagePath, image.URL)
 		if err := handleSSHError(out); err != nil {
-			return autoSetupInput{}, actionError{err: fmt.Errorf("failed to download image: %s %s %w", out.StdOut, out.StdErr, err)}
+			err := fmt.Errorf("failed to download image: %s %s %w", out.StdOut, out.StdErr, err)
+			conditions.MarkFalse(
+				s.scope.HetznerBareMetalHost,
+				infrav1.ProvisionSucceededCondition,
+				infrav1.ImageSpecInvalidReason,
+				clusterv1.ConditionSeverityError,
```

Comment:
this is only used up to now when there is an error that cannot be recovered. Is that the case here?

Reply:
Let's do the distinction between permanent and temporary error later.